privacyidea/static/components/dashboard/controllers/dashboardControllers.js

```diff
@@ -120,6 +120,10 @@ myApp.controller("dashboardController", function (ConfigFactory, TokenFactory,
         AuditFactory.get({"timelimit": "1d", "action": "*validate*", "success": "0"},
             function (data) {
                 $scope.authentications.fail = data.result.value.count;
+                $scope.authentications.users = Array();
+                angular.forEach(data.result.value.auditdata, function(auditentry){
```

Comment:
There are validate-check requests without user, but with a serial (e.g. token testing).

Those have no user or realm in the auditdata and are displayed with "@" in the dashboard. 

We could instead display and link the serial or exclude those entries. I think they are POST requests.

Reply:
done in c0547b7b